### PR TITLE
WIP: add locize script to header

### PR DESCRIPTION
**Pull request by youngfreezyVA**

### CODE REVIEW

File: src/site/includes/header.html [hunk 1]
```diff
@@ -83,6 +83,10 @@
     window.VetsGov.headerFooter = {{ headerFooterData }};
   </script>
 
+
+  <script id="locizify" projectid="e096f2b1-ae77-47fd-9ef1-122884a31231" debug="false" apikey="a24af10f-8178-4e8f-bac1-76aebb982d78" saveMissing="true" referencelng="en" fallbacklng="en" src="https://unpkg.com/locizify@^4.0.7"></script>
```
Comment (timwright12) [inline src/site/includes/header.html:87]: @youngfreezyVA Can we add and `async` attribute to this so it doesn't block page render?

Comment (youngfreezyVA) [reply src/site/includes/header.html:87]: yeah for sure, will do. bigger picture question for you @timwright12 .. do you know why my `yarn.lock` needs to be updated, and all the linting jobs are failing when this is the only change i made and am up to date with master?

I think people are committing with `--no-verify` and its blocking this PR from getting a review instance :(

Comment (timwright12) [reply src/site/includes/header.html:87]: Not completely sure, there's probably just a mismatch weirding out one of the tests TBH. I've seen it before and running `yarn` locally to update the lock file usually clears it out

Comment (youngfreezyVA) [reply src/site/includes/header.html]: yeah i ran `yarn` in this branch, but then the lockfile shows a diff against master... meaning the lockfile in master is wrong

Comment (timwright12) [reply src/site/includes/header.html]: it's ok if the lock files are different, they'll sync up on the next merge

Comment (youngfreezyVA) [reply src/site/includes/header.html]: ok, thank you!

**Review by timwright12 — APPROVED** [review]

**Pull request closed without merging**